src/ixdat/techniques/deconvolution.py

```diff
@@ -0,0 +1,254 @@
+"""Module for analysis of EC-MS measurements with impulse response and
+deconvolution of mass transport effects to obtain partial current densities."""
+
+from .ec_ms import ECMSMeasurement
+
```

Comment:
you've forgotten to import numpy and fft and matplotlib and whatever else you use.